.gitignore

```diff
@@ -24,7 +24,7 @@ docs/.jekyll-cache
 
 .vscode
 .DS_Store
-Manifest.toml
-test/Project.toml
+/Manifest.toml
```

Comment:
why switching to root level directory?

Reply:
True, we don't have to do that here. It's mostly useful for packages where the documentation is built with a specific reproducible set of packages which requires to add a `docs/Manifest.toml` file.

Reply:
I see - thanks. Maybe keep both?

Reply:
Do you mean both `Manifest.toml` and `/Manifest.toml`? `test/Project.toml` is needed for specifying the test environment.

Reply:
yes